mlir/lib/Target/LLVMIR/ModuleTranslation.cpp

```diff
@@ -1026,12 +1028,20 @@ static void addRuntimePreemptionSpecifier(bool dsoLocalRequested,
     gv->setDSOLocal(true);
 }
 
-/// Create named global variables that correspond to llvm.mlir.global
-/// definitions. Convert llvm.global_ctors and global_dtors ops.
-LogicalResult ModuleTranslation::convertGlobals() {
+/// Handle conversion for both globals and global aliases.
+///
+/// - Create named global variables that correspond to llvm.mlir.global
+/// definitions, similarly Convert llvm.global_ctors and global_dtors ops.
+/// - Create global alias that correspond to llvm.mlir.alias.
```

Comment:
This comment should be on the declaration. (I know that some parts of this file are outdated)